org.cznic.Datovka.yaml

```diff
@@ -0,0 +1,58 @@
+app-id: org.cznic.Datovka
+runtime: org.kde.Platform
+sdk: org.kde.Sdk
+runtime-version: 5.11
+command: datovka
+rename-desktop-file: datovka.desktop
```

Comment:
Can you try to get upstream to rename these files at least? The renaming is per the desktop/appstream spec.

Reply:
I'll talk to the upstream developers about it, it definitely makes sense to have the same app ID in flatpak and packages distributed in distros, but let's not stall the review on this.